client/src/components/admin/teams/TeamsView.jsx

```diff
@@ -51,8 +55,10 @@ class TeamsView extends Component {
 
   getCohortsList() {
     getAllCohorts().then(result => {
+      const studentsListByCohort = result.data.data.cohorts;
```

Comment:
I'm working on a PR to change the underlying graphQL api to specify getting only current cohorts.  This is fine now but it will be problematic when we have all cohorts available in the database.

Reply:
@magee I made the [changes](https://github.com/rpt-labs/ghostbuster/pull/57/commits/dbc3b60c4e914e980d9c2af5f60b0deb0e867070) to display only currently active cohorts in teams view. Please let me know if it looks ok